modules/tinymce/CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Added
+- It is now possible to bundle tinymce into a single js file. #TINY-10352
```

Comment:
Maybe we need so try to explain what we changed the release notes will likely be more detailed. Not sure if we should describe the internals of the css loading as separate `improved` item that is more similar to the one that you created. Something like this:

```suggestion
- CSS files are now also generated as separate JS files to improve bundling of all resources. #TINY-10352
- Added new `Resource.has` API that can be used to check if a resource has been loaded or not. #TINY-10352
- Added new `Resource.get` API that can be used to get a resource if it exists. #TINY-10352
- Added new `StylesheetLoader.loadRawCss` API that can be used to load CSS into a style element. #TINY-10352
- Added new `StylesheetLoader.unloadRawCss` API that can be used to unload CSS that was loaded into a style element. #TINY-10352
```